backend/src/main/java/capstone/storage/backend/item/models/Item.java

```diff
@@ -0,0 +1,11 @@
+package capstone.storage.backend.item.models;
+
+public record Item(String id,
+                   String name,
+                   String categoryName,
+                   String issuingCountry,
+                   String ean,
```

Comment:
Ein Vorschlag, mit Für und Wieder: Du könntest hier statt der einzelnen Attribute auch die Klasse `ItemResponse` wiederverwenden.

Vorteile:
* Weniger Code, weniger Herumkopieren von Eigenschaften, usw. (KISS)

Nachteile:
* Wenn sich die externe API ändert, dann ändert sich auch deine API (keine lose Kopplung)

Ich selbst sehe die Vorteile etwas überwiegen und würde es persönlich so vorschlagen:

```suggestion
                   ItemResponse item,
```